hca-api-stub/voa-rest.js

```diff
@@ -1,21 +1,83 @@
+const fs = require('fs');
 const loopback = require('loopback');
 const path = require('path');
 
+// TODO: node.js did not like `securityArtifacts` being declared as `let`. Why?
```

Comment:
yeah this is another version related issue, the support for let was hidden behind strict in the initial LTS release 4.2.x, and now that is not the case.